client/i18n/locales/english/intro.json

```diff
@@ -1346,6 +1338,20 @@
       }
     }
   },
+  "rosetta-code": {
+    "title": "Rosetta Code",
+    "intro": [
+      "Level up your creative problem solving skills with these free programming tasks from the classic Rosetta Code library.",
+      "These challenges can prove to be difficult, but they will push your algorithm logic to new heights.",
+      "<a href='https://rosettacode.org/wiki/Rosetta_Code' target='_blank' rel='noopener noreferrer nofollow'>Attribute: Rosetta Code</a>"
```

Comment:
On my end, the link is not showing up correctly which is weird because this looks like the same text from earlier and it works fine on the actual site 🤔 

<img width="951" alt="Screenshot 2023-12-19 at 6 26 33 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/55e78854-3433-4ef8-af86-3a70777faa98">